src/components/Header/notification.js

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { Dropdown } from 'react-bootstrap';
+import notificationImg from 'assets/images/notification.png';
+import { useSelector, useDispatch } from 'react-redux';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import NotificationArea from 'containers/Notification/notificationArea';
+import { getAllnotification, clearAllNotification } from 'store/actions/notification';
+
+import './style.scss';
+
+function HeaderNotification() {
+  const dispatch = useDispatch();
+  const allNotifications = useSelector((state) => state.notification);
+  const [notificationData, setNotificationData] = useState([]);
+
+  useEffect(() => {
+    if (allNotifications.notification.length === 0) {
+      dispatch(getAllnotification());
+    }
+  }, [allNotifications.notification.length, dispatch]);
```

Comment:
If there is no notification after get all notifications API call, do we call this every time?
I think it will send a lot of notification API call

Reply:
done